cmd/stack_package.go

```diff
@@ -493,8 +496,9 @@ func initialiseStackData(stackID string, stackImage string, stackYaml StackYaml)
 	return newStackStruct
 }
 
-func findStackAndRemove(log *LoggingConfig, stackID string, indexYaml IndexYaml) IndexYaml {
+func findStackAndRemove(log *LoggingConfig, stackID string, indexYaml IndexYaml) (IndexYaml, bool, error) {
```

Comment:
I don't believe you need to return an `error`, since we will always return `nil`.

Reply:
Yeah that was left over from a previous commit, removed it now cheers.